### PR TITLE
Change broken endpoint

### PR DESCRIPTION
**Pull request by Paulinakhew**
### WHY are these changes introduced?

- docs ask users to go to a route that doesn't exist

### WHAT is this pull request doing?

- fixed it ;) 

## Type of change

- [x] Patch: Bug (non-breaking change which fixes an issue)
- [ ] Minor: New feature (non-breaking change which adds functionality)
- [ ] Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above
- [ ] I have added/updated tests for this change
- [x] I have documented new APIs/updated the documentation for modified APIs (for public APIs)

### CODE REVIEW

**Review by JaKXz — APPROVED** [review]
🎉  Thanks Paulina!

File: docs/usage/oauth.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ app.get('/auth/callback', async (req, res) => {
 ```
 </details>
 
-After process is completed, you can navigate to `{your ngrok address}/oauth/begin` in your browser to begin OAuth. When it completes, you will have a Shopify session that enables you to make requests to the Admin API, as detailed next.
+After process is completed, you can navigate to `{your ngrok address}/auth/callback` in your browser to begin OAuth. When it completes, you will have a Shopify session that enables you to make requests to the Admin API, as detailed next.
```
Comment (paulomarg) [inline docs/usage/oauth.md:107]: I think we should actually be going to `/login` here, though?

Comment (thecodepixi) [reply docs/usage/oauth.md:107]: +1, the actual endpoint in the app should be `/login`, which will initiate the auth flow.

**Review by thecodepixi — APPROVED** [review]
Thanks, Paulina!

**Review by paulomarg — APPROVED** [review]

**Pull request merged**